infrastructure-live/dev/eu-central-1/data-stores/rds/terragrunt.hcl

```diff
@@ -0,0 +1,5 @@
+# This is the configuration for Terragrunt, a thin wrapper for Terraform: https://terragrunt.gruntwork.io/
+
+terraform {
```

Comment:
Shouldn't this `infra-live` stuff go into a `test` or `fixtures` or similar folder?

Reply:
It can go, but the action will run when push to `main` and open a Pull Request in this repo, e.g. https://github.com/gruntwork-io/patcher-action/pull/8. The diff from the PR is messy because I was testing it locally and it got the changes from the `js-action` branch.

For the time being, this will be the only example. We can provide richer examples when adding more docs on how to configure it against an infra-live repo.